exampleCourse/questions/element/code/question.html

```diff
@@ -18,6 +18,12 @@
       def square(x):
           return x * x
     </pl-code>
+
+    You can disable selection using the <code>prevent-select="true"</code> attribute.
+    <pl-code language="python" prevent-select="true" normalize-whitespace="true">
+      def square(x):
+          return x * x
+    </pl-code>
```

Comment:
If you really want to add this, remove `prevent-select` from the existing example below to avoid duplication.

Reply:
76fed5468032171d9808ceb2367427072140f5f6